qualtricssurvey/private/view.js

```diff
@@ -0,0 +1,11 @@
+function QualtricsSurveyView(runtime, element) {
```

Comment:
@stvstnfrd Do you mean not making changes to the Gruntfile at all? If I delete the `.js` file and leave the Gruntfile alone, Grunt will still generate an empty `.js` file. I need to get rid of the `jsView` section of the `concat` operation in order to stop that. Is that what you meant by updating the student_view or is that something else?